lib/ReplaceSource.js

```diff
@@ -18,16 +18,16 @@ class ReplaceSource extends Source {
 		this.replacements = [];
 	}
 
-	replace(start, end, newValue) {
+	replace(start, end, newValue, oldValue) {
```

Comment:
```
newValue => replacer || replacment
oldValue => replacee || replaced
```

(Nitpick)

Reply:
@michael-ciniawsky not related to this PR, here bug fix, not refactoring